configuration/src/main/java/pl/edu/uj/kimage/configuration/SettingName.java

```diff
@@ -0,0 +1,5 @@
+package pl.edu.uj.kimage.configuration;
+
+public enum SettingName {
+    NUMBER_OF_THREADS
```

Comment:
R U kiddin' me? 

What if we want to have properties per plugin? Even in the same file.